### PR TITLE
string operators 2.0

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Comparison operators
```
Comment (oldLady344) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:1]: all the operators should be in the code format.

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+* &ge;
+
+Greater than or equal to:
+
+```ngql
+nebula> YIELD 2 >= 2;
+[ERROR (-8)]: A string type can not be compared with a non-string type.
+```
+
+* &lt;
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:57]: `<=` not `&lt`

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+| -   | Change the sign of the argument   |
+
+Comparison operations result in a value of _true_ and _false_.
+
+* ==
+
+Equal. String comparisons are case-sensitive. Values of different types are not equal.
+
+```ngql
+nebula> YIELD 'A' == 'a';
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:24]: use `RETURN` in openCypher

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+| /   | Division operator   |
+| ==   | Equal operator   |
+| !=   | Not equal operator   |
+| <   | Less than operator   |
+| <=   | Less than or equal operator   |
+| -   | Minus operator   |
+| %   | Modulo operator   |
+| +   | Addition operator   |
+| *   | Multiplication operator   |
+| -   | Change the sign of the argument   |
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:15]: add two lines:
|`IS NULL` |  NULL test |
| `IS NOT NULL` | not NULL test |

and add example.

**Review by whitewum — COMMENTED** [review]
add a file in the `Data Types Chpater` named NULL.md

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:16]: add a note: the equality operator in nGQL is `==`. and in openCypher is `=`

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+---------
+```
+
+* &ge;
+
+Greater than or equal to:
+
+```ngql
+nebula> YIELD 2 >= 2;
+[ERROR (-8)]: A string type can not be compared with a non-string type.
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:54]: `"2" >= 2` ?

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md [hunk 7]
```diff
@@ -0,0 +1,62 @@
+# Logical operators
+
+|  **Name**    |  **Description**    |
+|:----|:----:|
+|   &&     |   Logical AND     |
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md:5]: I think `&&` had been changed to `AND` like.

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md [hunk 8]
```diff
@@ -0,0 +1,62 @@
...
+|   XOR      |   Logical XOR  |
+
+In nGQL, non-zero numbers are evaluated to _true_. For the precedence of the operators, refer to [Operator Precedence](8.precedence.md).
+
+* &&
+
+Logical AND:
+
+```ngql
+nebula> YIELD -1 && true;
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md:17]: RETURN

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md [hunk 9]
```diff
@@ -0,0 +1,62 @@
+# Logical operators
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md:1]: rename this file name and title to `Boolean operators`.

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/8.precedence.md [hunk 10]
```diff
@@ -0,0 +1,25 @@
...
+```
+
+For operators that occur at the same precedence level within an expression, evaluation proceeds left to right, with the exception that assignments evaluate right to left.
+
+The precedence of operators determines the order of evaluation of terms in an expression. To override this order and group terms explicitly, use parentheses.
+
+## Examples
+
+```ngql
+nebula> YIELD 2+3*5;
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/8.precedence.md:23]: RETURN

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/8.precedence.md [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+
+The following list shows the precedence of nGQL operators in descending order. Operators that are shown together on a line have the same precedence.
+
+```ngql
+- (negative number)
+!
+*, /, %
+-, +
+== , >=, >, <=, <, <>, !=
+&&
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/8.precedence.md:11]: AND OR

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/8.precedence.md (hunk 10 above)
Comment (Amber1990Zhang) [reply docs_2.0/manual_EN/4.ngql-guide/5.operators/8.precedence.md:23]: no

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md (hunk 3 above)
Comment (Amber1990Zhang) [reply docs_2.0/manual_EN/4.ngql-guide/5.operators/1.comparison.md:24]: better not

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md (hunk 8 above)
Comment (Amber1990Zhang) [reply docs_2.0/manual_EN/4.ngql-guide/5.operators/2.logical.md:17]: no

**Review by randomJoe211 — APPROVED** [review]

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/.2.boolean.md [hunk 12]
```diff
@@ -0,0 +1,62 @@
+# Boolean operators
+
+|  **Name**    |  **Description**    |
+|:----|:----:|
+|   AND     |   Logical AND     |
+|   !      |   Logical NOT     |
+|   \|\|   |   Logical OR   |
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/.2.boolean.md:7]: OR
NOT 
XOR

File: docs_2.0/manual_EN/4.ngql-guide/5.operators/.2.boolean.md [hunk 13]
```diff
@@ -0,0 +1,62 @@
...
+|   XOR      |   Logical XOR  |
+
+In nGQL, non-zero numbers are evaluated to _true_. For the precedence of the operators, refer to [Operator Precedence](8.precedence.md).
+
+* AND
+
+Logical AND:
+
+```ngql
+nebula> YIELD -1 && true;
```
Comment (whitewum) [inline docs_2.0/manual_EN/4.ngql-guide/5.operators/.2.boolean.md:17]: ditto.  use AND

**Review by whitewum — COMMENTED** [review]
minor

**Review by randomJoe211 — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**